distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -185,3 +185,8 @@ security.audit.roles=group,admin,manager,viewer,system-admin,system-history,syst
 # Disable Hazelcast version check
 #
 hazelcast.version.check.enabled=false
+
+#
+# Artemis Message Bus properties
+#
+artemis.amqp.port=5672
```

Comment:
SSL can be enabled on any port and the broker will enforce SSL when client connects to that port.